src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/branch/BitbucketCloudBranch.java

```diff
@@ -58,6 +65,20 @@ public String getName() {
         return name != null ? name : branch;
     }
 
+    public String getDate() {
+        return date;
+    }
+
+    @Override
+    public long getDateMillis() {
+        final SimpleDateFormat dateParser = new SimpleDateFormat("yyyy-MM-dd HH:mm:ssXXX");
```

Comment:
always going to be this format? I guess being cloud... not likely to change with localisation/prefs right?

Reply:
Same format as expected on [BitbucketCloudCommit.java](https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/master/src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/branch/BitbucketCloudCommit.java#L73) but instead of setting the timezone, I parse it.

Also defined at the [Bitbucket Cloud API docs](https://confluence.atlassian.com/bitbucket/repository-resource-1-0-296095202.html#repositoryResource1.0-GETlistofbranches)

Reply:
LGTM